### PR TITLE
Update config files, minify dist versions

### PR DESCRIPTION
**Pull request by ksquaredcoding**
Updated all the gam-head files and synchronized file naming conventions. Removed deprecated code and minified all gam-head files for dist folder

### CODE REVIEW

**Comment by ksquaredcoding** [comment]
[Update Prebid configuration for each website](https://trello.com/c/izq0ySl6/41-update-prebid-configuration-for-each-website)

File: foreverymom/index.html [hunk 1]
```diff
@@ -6,10 +6,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>ForEveryMom Ad Units</title>
 
-    <script type="text/javascript" src="https://ap.lijit.com/www/sovrn_beacon_standalone/sovrn_standalone_beacon.js?iid=13390662&amp;uid=241568" async></script>
     <script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
-    <script type="text/javascript" src="/foreverymom/scripts/prebid.min.js" defer></script>
+    <script type="text/javascript" src="/AdController.js"></script>
```
Comment (thechrise) [inline foreverymom/index.html]: @ksquaredcoding What is this AdController.js supposed to be? It looks like it's not included in the changes.

**Comment by thechrise** [comment]
It looks like we're missing Prebid in CL's gam-head.js file. Let's plan on adding that along with building out the ad slots in our advertising dashboards. We are missing a few opportunities throughout website configs.

**Review by thechrise — CHANGES_REQUESTED** [review]
It looks like we're missing Prebid in CL's gam-head.js file. Let's plan on adding that along with building out the ad slots in our advertising dashboards. We are missing a few opportunities throughout website configs.

**Pull request merged**